index.html

```diff
@@ -167,7 +167,35 @@ <h6>LET'S CONNECT</h6>
                             <p>skills</p>
                             <i class="fa fa-angle-right"></i>
                         </li>
-                    </ul>   
+                    </ul>  
+        </section>
+        <section class="contact">
+            <div class="contact-form">
+                <h2>Contact Me</h2>
+                <p>If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I'd love to help with it 
+                </p>
+                <form action="https://formspree.io/f/mdobaojq" method="POST">
+                    <ul class="form">
+                        <li class="form-content">
+                        <label for="name"></label>
+                        <input type="text" id="name" placeholder="Enter Your Name" maxlength="30" required>
```

Comment:
- [ ] Right here you are missing the `name` attribute. This is necessary in order to make formspree work. If you try to send a message, you will realize that it shows you an error message that says that the `name` attribute is necessary, so please make sure to implement it.